geoindicators/src/main/groovy/org/orbisgis/GenericIndicators.groovy

```diff
@@ -155,12 +155,13 @@ static IProcess weightedAggregatedStatistics() {
                     weights.each{weight, operations ->
                         // The operation names are transformed into upper case
                         operations.replaceAll({s -> s.toUpperCase()})
-                        if(operations.contains(AVG)) {
-                            weightedStdQuery += "b.weighted_avg_${var}_$weight,"
+                        if(operations.contains("AVG")) {
```

Comment:
You can keep `AVG` instead of `"AVG"`

Reply:
I did not see this change in the conflict. I modified the code in this way